zimui/src/plugins/videojs-ogvjs.js

```diff
@@ -0,0 +1,759 @@
+/* eslint-disable no-prototype-builtins */
+/**
+ * videojs-ogvjs
+ * @version 1.0.0
+ * @copyright 2023 Huong Nguyen <huongnv13@gmail.com>
+ * @license MIT
+ */
+import videojs from 'video.js'
```

Comment:
Please add a comment explaining:
- where this file has been retrieved from 
- since if I'm not mistaken this is a Github repo, also specify commit ID + date when this was retrieved

Goal is that it is clear for maintainers how to update this file, and when this is needed

Reply:
Added in [832ac24](https://github.com/openzim/youtube/pull/259/commits/832ac248db071e4a381764721beb9b8049c0786c)